changelog.yaml

```diff
@@ -16,7 +16,18 @@
 
 - version: "NEXT"
   date: TBD
-  
+  changes:
+    type: enhancement
+    impact: minor
+    title: Extend pipelinerun to be able to select networkpolicy
+    description: >-
+      Networkpolicy can be selected on pipelinerun now. See <a href="https://github.com/SAP/stewardci-core/tree/PR_profiles/docs/backend-api#fields-2">PipelineRun Fields</a>.

```

Comment:
This branch will be gone soon. You could link to the commit, but this might be gone as well, if we squash. Or you could already link to the master branch, although the change is not yet available there.

Reply:
Fixed